cbmc_viewer/symbol_table.py

```diff
@@ -101,9 +111,19 @@ def parse_pretty_name(sym):
     # Pretty name.: struct struct_name
     # Pretty name.: union union_name
 
-    name = sym.strip().split()[-1]
-    if re.match('^[a-zA-Z0-9_]+$', name):
-        return name
+    # Match the shape of the input format
+    line_match = re.match("^Pretty name\.: (.*)$", sym)
```

Comment:
Prefer to use the original construct modified to be sym.strip.split(":", 1)[-1]. This would also allow removing the following if statement.